### ISSUE TITLE
Proper PFAA Integration

### ISSUE THREAD

**Issue by Antillar** [Blood-Asp/GT5-Unofficial#1360]
GregTech has cross-mod compatibility between PFAA (Per Fabrica Ad Astra: Geoligica) for a very long time. The stone types and ores of PFAA can be processed by GregTech. However, this integration was never properly shaped. The materials only present in PFAA and unobtainable otherwise are always loded, slowing the loading time and creating problems for the new player who does not know of their absence in the world, as they show up in NEI, and furthermore they show up in Centrifuge/Electrolyzer recipes. And, as some of them contain very valuable materials, this may and does misguide the players when they try to figure their way through the mod.

My suggestion is to properly shape the cross-mod integration: adress the materials to PFAA and load them only if PFAA is present in the modpack. This effectively solves the problem of their appearance in recipes and players being misguided and trying to find them in the world in oreder to process. 

I have prepared the list of materials which have to be adressed to PFAA and arranged them in alphabetical order:
* Alunite
* Asbestos
* Basaltic Mineral Sand
* Borax
* Cassiterite Sand
* Chrysotile
* Diatomite
* Fullers Earth
* Garnet Sand
* Glauconite Sand
* Granitic Mineral Sand
* Gypsum
* Kaolinite
* Kyanite
* Mica
* Mirabilite
* Perlite
* Pollucite
* Pumice
* Quartz Sand
* Realgar
* Trona
* Vermiculite
* Volcanic Ashes
* Wollastonite
* Zeolite

All of these materials are totally unobtainable unless PFAA is present in the modpack.

**Comment by mura3277** [comment]
To add to this, these should also be disabled if PFAA is not installed.

**Comment by draknyte1** [comment]
Fullers earth is nice though, the rest are meh. (Only cause with FE things, the materials you get it from would then become relatively useless.)

**Comment by Antillar** [comment]
@Muramasa- yes, they should be disabled unless PFAA is installed. The purpose of this issue is to create PFAA relation to them.

**Comment by mura3277** [comment]
These materials are now disabled if PFAA is not installed, so this issue is for better PFAA support

**Comment by bHXDgiuMa5yLHyC8** [comment]
For real PFAA integration you may want to consider double checking that the ore spawning is balanced.

https://github.com/lawremi/CustomOreGen/issues/210

**Comment by Antillar** [comment]
@bHXDgiuMa5yLHyC8 the ore spawning is mostly balanced. As of the last build (pre4) the only material problem I have encountered is the absence of Borax, which was before a Rock Salt tertiary ore processing byproduct. All the ore spawning is balanced, at least with just pure GregTech and necessary mods. I am pretty sure every ore which is supposed to generate, does generate.
